src/users/__tests__/userHelpers-test.js

```diff
@@ -125,7 +124,7 @@ describe('getAutocompleteSuggestion', () => {
     const user7 = eg.makeUser({ name: 'Match App Normal', email: 'any3@example.com' }); // satisfy all conditions
     const user8 = eg.makeUser({ name: 'match', email: 'any@example.com' }); // duplicate
     const user9 = eg.makeUser({ name: 'Laptop', email: 'laptop@example.com' }); // random entry
-    const user10 = eg.makeUser({ name: 'Mobile App', email: 'any@match.com' }); // satisfy initials and email condition
+    const user10 = eg.makeUser({ name: 'Mobile App', email: 'any@match.com' }); // satisfy email condition
```

Comment:
nit: this and the remaining changes in this test should go in the first commit, together with the change in the code they describe

The commands I'd use for making this sort of revision -- pulling some changes forward from a later commit in a branch to an earlier one -- would be:
* `git rebase -i`; say to edit the first commit
* when the rebase reaches that commit, use `git checkout -p` to select the later changes to add.
  * For example, `git checkout -p main` if your local branch is `main`; or `git checkout -p typeahead-remove-diacritics` if you're using the same name locally as you have here.
  * In general you could also add the filename, or a directory containing it: `git checkout -p main -- src/users/__tests__/`. In this case there aren't a lot of other files to skip past, so you might not bother.
  * To get your bearings before doing the `git checkout -p`, you can use `git diff --stat -p -R main`. That will show you all the changes you'll get to choose from if you do `git checkout -p main`.

Reply:
It was just 2 or 3 changes in the comments, so I just edited the first commit manually and then the rest of the rebase was clean, but thanks for the info. I could definitely see it getting more complicated with larger changesets.

Reply:
Cool.

> I just edited the first commit manually and then the rest of the rebase was clean

Yeah, I forgot to mention: that's also how the `git checkout -p` approach works, except for the word "manually".

That is, it's just a way of editing what you have in the tree. The rebase doesn't know anything special about whether you edited with `git checkout -p` vs. with your editor or anything else.

But then when you continue the rebase and it goes to apply the later commit, when some of the hunks in that diff are already present, it doesn't complain, and instead just makes a commit using the remaining hunks.